lib/src/overlay_animation.dart

```diff
@@ -5,9 +5,12 @@ class _AnimatedOverlay extends StatefulWidget {
   /// [Duration.zero] means overlay display forever.
   final Duration duration;
 
-  /// The duration overlay show/hide's animation.
+  /// The duration overlay show animation.
   final Duration animationDuration;
 
+  ///overlay hide animation duration
```

Comment:
comment should start with uppercase and end with "." and "///" should also be followed by a space.

How about?
```suggestion
  /// The duration for overlay hide animation.
```